apiserver/facades/agent/secretsdrain/mocks/facade_mock.go

```diff
@@ -0,0 +1,520 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: github.com/juju/juju/apiserver/facade (interfaces: Context,Authorizer)
+
+// Package mocks is a generated GoMock package.
+package mocks
```

Comment:
We don't really need mocks sub-packages. They were only ever introduced to get around circular dependencies due in part to having separate {package} and {package}_test packages. Just stick it in _package_mock_test.go_